woocommerce/woocommerce-accommodation-bookings
# Update dependencies

### All Submissions:


### Changes proposed in this Pull Request:

Update our npm dependencies. This PR replaces the following PRs that were opened by dependabot:

- #277 
- #279 
- #280 
- #282 
- #286 
- #287 

### How to test the changes in this Pull Request:

1. Ensure you are running node v10 and npm v6 or below, as the way we are building assets is no longer supported as of [npm 7](https://github.com/npm/cli/blob/latest/changelogs/CHANGELOG-7.md#all-lifecycle-scripts). No changes to this were made in this PR, this is existing functionality, but would be nice to modernize this
2. On a clean install, run `npm install`
3. Then run `npm run build:dev`
4. Ensure a single CSS file is created properly, a single minified JS file is created properly and a single POT file is created
5. Test that any functionality that relies on the CSS or JS files still works as intended

### Other information:


### Changelog entry

> Update all npm dependencies.
